server.py

```diff
@@ -87,6 +95,8 @@ def decrypt():
         return client_error("Decryption Failure")
     except binascii.Error:
         return client_error("Request payload was not base64 encoded")
+    except InvalidTokenException as e:
+        return client_error(str(e))
```

Comment:
Do you need to add an ERROR level log line here? Either `logger.exception` or `logger.error`.